### PR TITLE
Adds tiny fans to supply shuttle(s)

### PR DESCRIPTION
**Pull request by Agameofscones**
## About The Pull Request

Adds tiny fans (prevents atmos loss) onto cargo ships. While the cargo techs may leave the blast doors open and vent cargobay atmos, the cargo ship itself won't be depressurized from negligence anymore.

## Why It's Good For The Game

Idiot proofs the cargo shuttle. Quality of life improvement.

image
(On each blast door & airlock)

## Changelog
:cl: Triiodine
add: Added tiny fans to the Supply Shuttle blast doors and airlocks.
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Fix the errors

**Comment by Toxici11i** [comment]
nopls tiny fans stop atmos changes on both sides and add air passively

File: _maps/shuttles/cargo_box.dmm [hunk 1]
```diff
@@ -47,10 +48,8 @@
 	name = "Supply Shuttle Airlock";
 	req_access_txt = "31"
 	},
-/obj/docking_port/mobile/supply{
-	dwidth = 5;
-	width = 12
```
Comment (qwertyquerty) [inline _maps/shuttles/cargo_box.dmm:52]: Why did you remove these.

Comment (Agameofscones) [reply _maps/shuttles/cargo_box.dmm:52]: StrongDMM removed them when saving back to TGM. Checking the  map post-save in alternate editors (FastDMM & Byond's built in editor) reveals that the values remain the same despite removal. Checkout the branch and check yourself. It compiles without issue and works ingame.

I have no idea 🤷‍♂ Default values probably?

**Comment by Agameofscones** [comment]
> nopls tiny fans stop atmos changes on both sides and add air passively

What object could I use instead? Admittedly I'm not all that familiar with TG based atmospherics.

File: _maps/shuttles/cargo_box.dmm (hunk 1 above)
Comment (ike709) [reply _maps/shuttles/cargo_box.dmm:52]: StrongDMM does remove default values IIRC.

**Comment by ike709** [comment]
@Toxici11i Yeah that's the point.

**Comment by ike709** [comment]
Travis is being weird but it did pass.
